recipes/phables/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "phables" %}
-{% set version = "1.4.0" %}
+{% set version = "1.4.1" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://github.com/Vini2/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
-  sha256: 09b843141ab32cda5016741b7f31ccefcb659448f0fd4aa55697c509680e16e9
+  sha256: f456260bf710073986ecd524975e09083c5cf7d0ad3a0e6808ba48a2b7dc0ee4
 
 build:
   number: 0
-  noarch: python
+  noarch: false
```

Comment:
What is the reason for this change ?
I see no arch specific binaries built by the recipe: https://github.com/bioconda/bioconda-recipes/pull/51124#issuecomment-2390765305

Reply:
I only wanted to skip the Windows build as an [environment](https://github.com/Vini2/phables/blob/develop/phables/workflow/envs/smg.yaml) created in phables uses [fraggenescan](https://anaconda.org/bioconda/fraggenescan) and [hmmer](https://anaconda.org/bioconda/hmmer) which only support linux and osx.

Reply:
Normally you would use the [`build/skip` entry](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds) if you want to skip windows.

Reply:
Though it does also say that if you shouldn't use a preselector (e.g., `# [win]`) with `noarch` packages

Reply:
Bioconda does not build for Windows anyway. Only Linux and OSX are supported at the moment.

Reply:
So what would be the best way to specify this in the recipe?

Reply:
Honestly I do not see a better solution than the current one.
You want to avoid Windows users to try to install the package, so you dropped `noarch`.

If you decide to go back to `noarch` then you can tell the users that Bioconda does not support Windows. It is written at the very top of the home page (https://bioconda.github.io/)